### PR TITLE
better UX

### PR DESCRIPTION
**Pull request by TomasTomecek**
* Probe + is_port_open = wait_for_port ♡
* docstring for is_port_open
* better UX for run_via_*
* more logging
* increase default timeouts to 10 seconds
* nicer and easier imports for users

### CODE REVIEW

File: conu/backend/docker/container.py [hunk 1]
```diff
@@ -167,15 +169,32 @@ def get_ports(self):
             ports.append(p.split("/")[0])
         return ports
 
-    def is_port_open(self, port, timeout=2):
+    def is_port_open(self, port, timeout=10):
```
Comment (dhodovsk) [inline conu/backend/docker/container.py]: Maybe 10 seconds is too much for yes/no question type of method.

File: conu/backend/docker/container.py [hunk 2]
```diff
@@ -167,15 +169,32 @@ def get_ports(self):
...
+        """
+        check if given port is open and receiving connections
+
+        :param port: int
+        :param timeout: int, how many seconds to wait for connection; defaults to 2
```
Comment (dhodovsk) [inline conu/backend/docker/container.py]: It now defaults to 10

File: conu/backend/docker/container.py (hunk 1 above)
Comment (TomasTomecek) [reply conu/backend/docker/container.py]: agreed

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/SnNGZ](https://da.gd/SnNGZ)__
* Comment: __This is testing CI__

**Pull request merged**